Manifest.toml

```diff
@@ -0,0 +1,258 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
Manifest.toml should never be committed: first remove it, then add it to `.gitignore`